### PR TITLE
[zh] sync and update statefulset.md

### PR DESCRIPTION
**Pull request by yangjiao2**
update based on English version of statefulset.md

1. added a block inside #stable-network-id
2. sync tables and blocks
3. added a sentence in #parallel-pod-management

welcome all the comments :)

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/statefulset.md [hunk 1]
```diff
@@ -217,9 +227,46 @@ StatefulSet 可以使用 [无头服务](/zh/docs/concepts/services-networking/se
...
+取决于集群域内部 DNS 的配置，有可能无法查询一个刚刚启动的 Pod 上的 DNS 命名。这个情况会在其他集群域内部的客户端在主机 Pod 创建完成前
+发出查询请求。负缓存 (通常来说在 DNS 内) 意味着之前失败的查询结果在特定秒数内被记录和被重用，甚至直至 Pod 正常运行。
+
+如果需要及时查询创建之后的 Pod ，有以下选项：
+
+- 直接查询 Kubernetes API（比如，利用 watch 机制）而不是依赖于 DNS 查询
+- 减少 Kubernetes DNS 提供商的 缓存时效（通常这意味着修改 CoreDNS 的 config map，目前 config map 会缓存30秒信息）
+
+正如 [限制](#limitations) 中提到的，创建 [无头服务](zh/docs/concepts/services-networking/service/#headless-services)
+需要对其 Pod 的网络标识负责。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/statefulset.md:257]:
```suggestion
以便为 Pod 提供网络标识。
```

File: content/zh/docs/concepts/workloads/controllers/statefulset.md [hunk 2]
```diff
@@ -146,15 +146,25 @@ spec:
...
+
 -->
+上述例子中：
+
 * 名为 `nginx` 的 Headless Service 用来控制网络域名。
 * 名为 `web` 的 StatefulSet 有一个 Spec，它表明将在独立的 3 个 Pod 副本中启动 nginx 容器。
 * `volumeClaimTemplates` 将通过 PersistentVolumes 驱动提供的
   [PersistentVolumes](/zh/docs/concepts/storage/persistent-volumes/) 来提供稳定的存储。
 
+StatefulSet 的命名需要遵循 [DNS 子域名](zh/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)。
```
Comment (yangjiao2) [inline content/zh/docs/concepts/workloads/controllers/statefulset.md:166]:
```suggestion
StatefulSet 的命名需要遵循[DNS 子域名](zh/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)规范。
```

Comment (tengqm) [reply content/zh/docs/concepts/workloads/controllers/statefulset.md]: No. The extra space here is needed. It is between Chinese and English characters which happen to be link texts.

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**